i18n/en.po

```diff
@@ -583,12 +583,21 @@ msgstr ""
 "you just registered and repeat the process up a maximum of ${ maxAttempts } "
 "times.</p>"
 
-#: client/components/status/status.js:375
+#: client/components/login/login.js:201
+#: client/components/status/status.js:378
 #. wait to trigger login to avoid getting stuck
 #. in captive portal firewall rule reloading
 msgid "PLEASE_WAIT"
 msgstr "Please wait..."
 
-#: client/components/status/status.js:377
+#: client/components/status/status.js:380
 msgid "PLEASE_LOGIN"
 msgstr ""
+
+#: client/components/registration/registration.js:870
+msgid "CONFLICT_ORGS"
+msgstr "This user is registered with other organizations:"
+
+#: client/components/registration/registration.js:878
+msgid "CONFLICT_SIGNIN"
+msgstr "Do you want sign in to this organization?"
```

Comment:
Do you own the account mentioned above and want to sign in with this organization?

Choosing yes will take you to the log in form where you can log in to this service using your existing credentials.